src/ophys_etl/transforms/trace_transforms.py

```diff
@@ -0,0 +1,47 @@
+from typing import List
+
+import numpy as np
+from scipy.sparse import coo_matrix
+
+
+def extract_traces(movie_frames: np.ndarray, rois: List[coo_matrix],
```

Comment:
Reason for not importing from AllenSDK instead?

Reply:
I am concerned about duplicating/slightly changing behavior that's currently in the AllenSDK. We should keep the relationship between this repo and the SDK in mind as we continue development.

Reply:
The problem I encountered was that the AllenSDK extract traces does not appear to be able to handle the coo_matrix format (and also can't deal with weighted masks). I wanted to get an apples to apples comparison of the traces between weighted/binarized traces.

I can add a note that the version in this repo should be removed if we end up going with the allensdk version?

Reply:
I recommend just keeping it as-is right now. We should have a discussion in the future about the role of the AllenSDK moving forward, now that we are abstracting more internal pipelines away (`neuron_morphology` is another example).

Reply:
I've added a note to the `extract_traces` implementation in this repo!